src/edgepi/calibration/edgepi_eeprom.py

```diff
@@ -288,7 +310,7 @@ def init_memory(self):
             self.data_list = json.loads(mem_content)
             self.used_size = mem_size
         # Memory Empty
-        elif mem_size == 0xFFFF:
+        elif mem_size == EdgePiMemoryInfo.FACTORY_DEFAULT_VALUE.value:
```

Comment:
why don't you check 

```
if mem_size == EdgePiMemoryInfo.FACTORY_DEFAULT_VALUE.value
```

first?

This way you don't need to repeat `mem_size != EdgePiMemoryInfo.FACTORY_DEFAULT_VALUE.value:` in the previous if statements for "Memory Full" and "mem_size of greater than EdgePiMemoryInfo.USER_SPACE_MAX.value"

Reply:
97a9ec241e7eae9d3d21191a6fe48cc80ac44a09

good point, fixed